build.xml

```diff
@@ -734,8 +734,8 @@
       <!-- each dependency set then defines the subset of the dependencies for that dependency set -->
       <artifact:pom id="build-deps-pom"
                     artifactId="cassandra-build-deps">
-        <parent groupId="org.apache.cassandra"
-                artifactId="cassandra-parent"
+        <parent groupId="com.datastax.dse"
```

Comment:
it looks like our group it could be in some property

Reply:
True, but I wanted to keep the same structure as the original build.xml to minimize chances of conflicts when merging upstream

Reply:
well, comparing to this change set you would have just one extra line where the property is defined